eng/testing/tests.wasm.targets

```diff
@@ -124,17 +123,17 @@
       <WasmDebugLevel Condition="'$(DebuggerSupport)' == 'true' and '$(WasmDebugLevel)' == ''">-1</WasmDebugLevel>
     </PropertyGroup>
 
-    <ItemGroup>
-      <WasmSatelliteAssemblies Include="$(PublishDir)*\*.resources.dll" />
-      <WasmSatelliteAssemblies CultureName="$([System.IO.Directory]::GetParent('%(Identity)').Name)" />
+    <ItemGroup Condition="'$(IncludeSatelliteAssembliesInVFS)' == 'true'">
+      <_SatelliteAssemblies Include="$(PublishDir)*\*.resources.dll" />
+      <_SatelliteAssemblies CultureName="$([System.IO.Directory]::GetParent('%(Identity)').Name)" />
+      <_SatelliteAssemblies TargetPath="%(CultureName)\%(FileName)%(Extension)" />
 
-      <WasmAssembliesToBundle Include="$(PublishDir)\*.dll"/>
+      <WasmFilesToIncludeInFileSystem Include="@(_SatelliteAssemblies)" />
+    </ItemGroup>
 
+    <ItemGroup>
+      <WasmAssembliesToBundle Include="$(PublishDir)\**\*.dll"/>
```

Comment:
Is this correct? Shouldn't this use output from linker (it could also solve the problem with invariant mode). ?

Reply:
This is what we have been doing, but I will be changing this in an upcoming PR to use output from the build/publish. This works fine, because we run after the `publish` target, by which time the linker output has already been copied to `publish` dir.

re:invariant mode problem, this was because msbuild was being run in invariant mode, so it couldn't validate the culture names, and would mark the resources as being culture neutral. I'll add this info to the description.